7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    N = morphology_data.shape[0]
</pre><p>I think you define N twice here. If you want to show they are equal consider adding a print or assert statement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yes, thank you. I removed that line now.</p>